README.md

```diff
@@ -1592,6 +1592,36 @@ impl MyRecord {
 }
 ```
 
+To reduce boilerplate code, one can also achieve this using the `format` attribute within `#[derive(Tabled)]`.
```

Comment:
Thanks for README update.

note: could be also updated in `tabled` documentation itself.